terraform/backend_uk.tf

```
@@ -0,0 +1,147 @@
+module "backend_uk" {
+  source = "git@github.com:trade-tariff/trade-tariff-platform-terraform-modules.git//aws/ecs-service?ref=aws/ecs-service-v1.2.0"
```

```suggestion
  source = "git@github.com:trade-tariff/trade-tariff-platform-terraform-modules.git//aws/ecs-service?ref=aws/ecs-service-v1.4.0"
```

I made some changes while debugging the module yesterday afternoon and I've now released a new version of the module that works with Fargate correctly: [`aws/ecs-service-v1.4.0`](https://github.com/trade-tariff/trade-tariff-platform-terraform-modules/releases/tag/aws%2Fecs-service-v1.4.0)